RELEASE.md

```diff
@@ -100,6 +100,10 @@ Go to  https://github.com/prometheus-operator/prometheus-operator/releases/new,
 
 For patch releases, submit a pull request to merge back the release branch into the `master` branch.
 
+## Update website
+
+Bump the operator's version in the [website](https://github.com/prometheus-operator/website/blob/main/data/prometheusOperator.json) repository.
```

Comment:
This should be automated. Maybe we can reconfigure the site to check what is the latest version of the operator? 

/cc @metalmatze

Reply:
It would be ideal. The PR goal is to at least document the manual step until we have automation.

Reply:
Yup, I mean if we can figure out a simple way to get the latest version, then that would be even nicer. I figured that it might not be worth the overhead considering it would be based on some heuristic.

Reply:
We have an always up-to-date version in `https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/master/VERSION`. It is a matter of having the site do a request to that endpoint and use the content for setting the version value.